doc/examples/sched.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: nodetopology.openshift.io/v1alpha1
 kind: NUMAResourcesScheduler
 metadata:
-  name: topo-aware-scheduler
+  name: numaresourcesscheduler
```

Comment:
not sure we want to enforce this name, on one hand this is likely to clash with the name we will use when we switch to SSOP. OTOH, this seems also a fair thing to change.

Reply:
IIRC, SSOP using the ```secondary-scheduler``` name for its CR

Reply:
let's keep like this for now; the biggest reason is we gave up to be 1:1 compatible with SSOP, so no point in using the same name: it only adds confusion.